Makefile.toml

```diff
@@ -1352,5 +1355,56 @@ rm -rf ${BUILDSYS_STATE_DIR}
 '''
 ]
 
+[tasks.install-testsys]
+dependencies = ["setup", "fetch-sources"]
+script = [
+    '''
+    cargo install \
+      ${CARGO_MAKE_CARGO_ARGS} \
+      --path tools/testsys \
+      --root tools \
+      --force \
+      --quiet
+    '''
+]
+
+[tasks.setup-testsys]
+dependencies = ["setup", "fetch-sources"]
+script = [
+    '''
+    set -eu
+    export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+    testsys install
+    '''
+]
```

Comment:
Install is currently idempotent _ish_ . Most of the time it is, but sometimes it will throw and error. More research is needed to determine why. Can we add an issue for this and move forward?